packages/experimental/kai/src/playwright/app-manager.ts

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import type { Context as MochaContext } from 'mocha';
+import type { ConsoleMessage, Page } from 'playwright';
+
+import { sleep, synchronized, Trigger } from '@dxos/async';
+import { setupPage } from '@dxos/test';
+
+// TODO(wittjosiah): Get this from executor.
+const BASE_URL = 'http://localhost:4200';
+
+export class AppManager {
```

Comment:
Is the name a pattern? Otherwise does this represent the browser context or the user, etc?

Reply:
it's not quite a pattern yet, but starting to become one. I think browser context/profile is one-to-one with a user/device so I'd say yes to both.